src/test/webhook.spec.ts

```diff
@@ -223,19 +225,31 @@ describe("Webhook testing", function () {
         });
 
       sinon.assert.calledOnce(sendWebhookNotificationSpy);
+
       sinon.assert.calledWith(
         sendWebhookNotificationSpy,
         userId,
         "create_recipe",
         response.body
       );
+
+      response.body.event = "create_recipe";
```

Comment:
why are you overwriting response body field?

Reply:
@MoodupDev because response.body contains recipe and axios.post is called with object containing recipe and event fields.